src/deluge/model/scale/preset_scales.h

```diff
@@ -13,7 +13,7 @@
  * See OFFSET_6|5_NOTE_SCALE for how to deal with that.
  *
  * The actual definitions expand this macro, picking the thing they need:
- * - enum PresetScales
+ * - enum PresetScale
```

Comment:
should this just be scale?

Reply:
Yes, it should. I must have updated from wrong branch! Traveling now, will fix later today or tomorrow.

Is force push ok once I find the right commits, or do you prefer a fixup commit?

Reply:
So, in principle yes, and I would like to do that, but we get some conflicts with `deluge::gui::ui::keyboard::controls::ChordModeChord` when doing that.

I'll look into making either one an enum class in a separate PR, if that's ok.